.env.sample

```diff
@@ -37,3 +37,9 @@ EMAIL_UPTIME_NOTIF_TEMPLATE_ID=2
 # Cron like sintax
 UPTIME_NOTIF_RECURRENCE=0 * * * *
 
+# Swagger settings
+
+# Swagger docs default port
+SWAGGER_DOCS_PORT=8080
+# Swagger docs default url
+SWAGGER_DOCS_URL=/api/swagger
```

Comment:
How do I define location of hactar-backend for api calls. From what I see this just define path on which swagger-ui is exposed and port. But by default swagger ui will invoke api on his root which when deployed won't be same as hactar-backend url